### PR TITLE
Define Invited Individual term and rename policy

### PR DESCRIPTION
**Pull request by foolip**
Goes with https://github.com/whatwg/participate.whatwg.org/pull/170.

### CODE REVIEW

**Comment by foolip** [comment]
Note that this doesn't *actually* define the term now, but if we want to do this it should be defined here rather than in https://github.com/whatwg/participate.whatwg.org/pull/170.

**Comment by foolip** [comment]
I've now updated https://github.com/whatwg/participate.whatwg.org/pull/170 to reference the term defined here, so this is ready for review.

File: Invited Individual Policy.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
 6. Once all [Steering Group Member]s have indicated support, and the scope of the [Requestor]’s
    [Workstream] participation is understood, the [Requestor]’s information
    is added to a database, and the [Requestor] is invited to sign the
    [Contributor and Workstream Participant Agreement] as an [Individual] with
-   [Workstream Participant Invitation] status.
+   [Invited Individual] status.
 
 ## Steering Group decision considerations
 
 The [Steering Group] reserves the right to approve or reject a request for a
```
Comment (annevk) [inline Invited Individual Policy.md:40]:
```suggestion
The [Steering Group] reserves the right to approve or reject a request for
```

**Review by annevk — APPROVED** [review]
Looks good modulo nit. We should remember to also add a redirect to whatwg/whatwg.org.

**Review by travisleithead — APPROVED** [review]
This is a great readability improvement to match the terminology the SG would like to use for this term in the Agreement.

**Comment by annevk** [comment]
The one thing still missing here is a corresponding PR to add the redirects.

**Review by tantek — APPROVED** [review]
LGTM, also ok with any requested clean-up / editorial changes.

**Comment by annevk** [comment]
(My comments above about the redirect are off base. It seems we preserve the URL on `whatwg.org`.)

**Pull request merged**